### DISCUSSION TITLE
Can the editor only be used locally?

### DISCUSSION THREAD

**Discussion by leo-petrucci** [tinacms/tinacms#3202] [Help] [answered]
I'm looking to an alternative to the soon-dead netlify-cms and I stumbled upon TinaCMS.

From the Docs I seem to understand that the editor is only accessible during development, while for production users have to use Tina Cloud. Is this correct? 

I couldn't get a straight answer from the docs so I thought I'd come ask here.

**Comment by jamespohalloran** [comment, marked as answer]
That's correct! We're strongly considering a [self hosted Tina backend in the future](https://tina.io/docs/self-hosted/overview/), but still working through the details of that.

**Reply by leo-petrucci** [reply]
Aw that's unfortunate, thank you for the help!
